addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1080,6 +1080,31 @@ registry._fixAppleCollapse = Animation.extend({
     },
 });
 
+registry.anchorSlide = Animation.extend({
+    selector: 'a[href^="/"][href*="#"]',
+    events: {
+        'click': '_onAnimateClick',
+    },
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     */
+    _onAnimateClick: function (ev) {
+        var hash = window.location.hash;
+        var $anchor = $(hash);
+        if ($anchor.length && this.$target[0].pathname === window.location.pathname) {
```

Comment:
Comment remains: would not it be better to target URL changes instead of link clicks ? There are ways to listen to hash changes I think ( https://caniuse.com/#search=hashchange, using jQuery). That way no need to check the pathname and add a click handler on every link, juste creating an animation targeting the #wrapwrap element and listening to this event on the window should do trick.

Reply:
I tried to handle it in start function with selector #wrapwrap but it doesn't seem to behave like expected. It detects hash change but .animate() isn't working as expected (buggy). What am I missing ?

```suggestion
start: function () {
    $(window).on('hashchange', function (ev) {
        console.log('hashchange');
        var hash = window.location.hash;
        var $anchor = $(hash);
        if ($anchor.length) {
            ev.preventDefault();
            $('html, body').animate({
                scrollTop: $anchor.offset().top,
            }, 500);
        }
    });
},
```

Reply:
@kea14 Then I suppose the body scrolls then the event is fired, so bad idea.

Anyways, please try testing and make sure everything is handled and thought through before sending to review, I easily found two things which are not ok... 
- If the anchor is put in the page then the user tries to make a link to it (most common user case), you cannot. Because $.get will render the not-yet-saved page and thus not found the new anchor... you must have seen this during development !
- First click on an anchor does not scroll with animation.